requirements/tools.txt

```diff
@@ -112,4 +112,4 @@ wrapt==1.11.2             # via astroid, deprecated
 zipp==0.5.2               # via importlib-metadata
 
 # The following packages are considered to be unsafe in a requirements file:
-# setuptools==41.0.1        # via ipython, safety, sphinx, twine
+# setuptools==41.1.0        # via ipython, safety, sphinx, twine
```

Comment:
:grimacing: 

Can you cherry-pick 14bc3e9ff8a5690763480458065eb7b16f2557bb onto this branch?  That should make the dependency checker in CI happy again...  Sorry!

Reply:
@Zac-HD I've included the changes rather than cherry picking the commit, should look good if you want to check `hypothesistooling/__main__.py` in the latest version of the branch.